app/controllers/users/sessions_controller.rb

```diff
@@ -22,9 +22,21 @@ def timeout
     end
 
     def destroy
-      # TODO: log out of IdP using OIDC logout endpoint
-      #   https://developers.login.gov/oidc/#logout
-      super
+      if session[:id_token]
+        logout_request = self.class.logout_utility.build_request(
+          id_token: session[:id_token],
+          post_logout_redirect_uri: 'http://localhost:3001/'
+        )
+        sign_out(current_user)
+        redirect_to(logout_request[:redirect_uri]) and return
```

Comment:
since this is the last line in the branch, and there's no other code, the `and return` is redundant right?

Reply:
Also can we use the property name? `logout_request.redirect_uri`

Reply:
Thanks for catching that. I was having issues with ordering of signing out/redirect and the `and return` was necessary if you wanted to do something after a redirect, but the if/else moved things around such that it wasn't needed.